README.md

```diff
@@ -124,11 +124,11 @@ This repository is a repository template, which means you can use the `Use this
 
 ### Make it yours
 
-You will now want to make this project yours by replacing all `stator` occurrences with your own project name.
+You will now want to make this project yours by replacing all organization and project naming occurrences with your own names.
 Thankfully, we have a script just for that:
 
 ```
-npm run rename-project {YOUR_PROJECT_NAME}
+npm run rename-project -- --organizationName {YOUR_ORGANIZATIO_NAME} --projectName {YOUR_PROJECT_NAME}
```

Comment:
organization*

Reply:
since its already called rename-project maybe --organization is enough, same for --project